base/system/diskpart/lang/zh-CN.rc

```diff
@@ -136,8 +137,8 @@ END
 /* Common Error Messages */
 STRINGTABLE
 BEGIN
-    IDS_ERROR_MSG_NO_SCRIPT "错误开始的脚本: %s\n"
-    IDS_ERROR_MSG_BAD_ARG "错误处理参数: %s\n"
+    IDS_ERROR_MSG_NO_SCRIPT "打开脚本时出错: %s\n"
+    IDS_ERROR_MSG_BAD_ARG "处理参数时出错: %s\n"
```

Comment:
```suggestion
    IDS_ERROR_MSG_NO_SCRIPT "打开脚本时出错：%s\n"
    IDS_ERROR_MSG_BAD_ARG "处理参数时出错：%s\n"
```

Reply:
I really left a lot of these symbols out... causing lots of inconsistencies...